templates/kernel/index.html

```diff
@@ -137,4 +137,31 @@ <h2 id="custom-kernels">Optimised kernels</h2>
       </p>
     </div>
   </section>
+
+  <section class="is-active p-takeover--dark js-takeover is-dark">
+    <div class="row u-equal-height">
+      <div class="col-6 col-medium-3 u-vertically-center">
+        <p class="p-text--small-caps">Whitepaper</p>
+        <h2 class="p-heading--1 u-no-padding--top">
+          A CTO's guide to real-time kernel
+        </h2 class="p-heading--1">
+        <h3 class="p-heading--4">Understanding real-time systems, their cases and inner workings</h3>
+        <div class="u-hide--small">
+          <p>
+            <a href="/engage/cto-guide-real-time-kernel?utm_source=kernelpage"
+               class="p-button--positive u-no-margin--bottom">Download <span class="u-off-screen">the RTF guide</span> now</a>
+          </p>
+        </div>
+      </div>
+      <div class="col-6 col-medium-3 u-vertically-center u-align--center u-align--left-medium u-hide--small">
```

Comment:
We should probably only hide this one on small screens. There's not way to download it on small screens if we hide both the download button and asset

Reply:
Thanks, not sure what was going on in this one :sweat: